README.md

```diff
@@ -1 +1,440 @@
+                .apiKey(logdashApiKey)
+                .enableConsoleOutput(false) // Use Spring's logging
+                .build();
+    }
+}
+
+@RestController
+public class UserController {
+
+    private final LogdashLogger logger;
+    private final LogdashMetrics metrics;
+
+    public UserController(Logdash logdash) {
+        this.logger = logdash.logger();
+        this.metrics = logdash.metrics();
+    }
+
+    @GetMapping("/users/{id}")
+    public User getUser(@PathVariable Long id) {
+        logger.info("Fetching user", Map.of("userId", id));
+        metrics.increment("user.fetch.requests");
+
+        var user = userService.findById(id);
+
+        logger.debug("User retrieved", Map.of(
+                "userId", id,
+                "username", user.getUsername()
+        ));
+
+        return user;
+    }
+}
+```
+
+### Standalone Applications
+
+```java
+public class StandaloneApp {
+    public static void main(String[] args) {
+        var logdash = Logdash.builder()
+                .apiKey(System.getenv("LOGDASH_API_KEY"))
+                .build();
+
+        var logger = logdash.logger();
+        var metrics = logdash.metrics();
+
+        // Add shutdown hook for graceful cleanup
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            logger.info("Application shutting down");
+            logdash.close();
+        }));
+
+        // Your application logic
+        runApplication(logger, metrics);
+    }
+}
+```
+
+## Logging
+
+### Log Levels & Context
+
+```java
+var logger = logdash.logger();
+
+// Simple messages
+logger.error("Database connection failed");
+logger.warn("High memory usage detected");
+logger.info("User session started");
+logger.http("GET /api/health - 200 OK");
+logger.verbose("Loading user preferences");
+logger.debug("Cache hit for key: user_123");
+logger.silly("Detailed trace information");
+
+// Structured logging with context
+logger.info(
+    "Payment processed",
+    Map.of(
+        "userId", 12345,
+        "amount", 29.99,
+        "currency", "USD",
+        "paymentMethod", "credit_card",
+        "transactionId", "txn_abc123"
+    )
+);
+
+logger.error(
+    "API call failed",
+    Map.of(
+        "endpoint", "https://external-api.com/data",
+        "statusCode", 503,
+        "retryAttempt", 3,
+        "errorMessage", "Service temporarily unavailable"
+    )
+);
+```
+
+## Metrics
+
+Track business and technical metrics to monitor application performance:
+
+```java
+var metrics = logdash.metrics();
+
+// Counters - track events and occurrences
+metrics.increment("page_views");
+metrics.increment("api_requests", 5);
+metrics.increment("user_signups");
+
+// Gauges - track current values
+metrics.set("active_users", 1_250);
+metrics.set("memory_usage_mb", 512.5);
+metrics.set("queue_size", 0);
+
+// Changes - modify existing values
+metrics.change("response_time_ms", -100);
+metrics.change("error_rate", 0.05);
+metrics.decrement("available_licenses", 1);
```

Comment:
Please make it same as in other SDKs. So the contract for metrics is only `set` or `mutate`.